### DISCUSSION TITLE
Warning: Invalid DOM property

### DISCUSSION THREAD

**Discussion by vprada25** [ant-design/ant-design#47305] [General]
![image](https://github.com/ant-design/ant-design/assets/41810525/8cb76fc4-d3ab-426c-93f9-8d3848dbbd26)
how can I solve this warning?

**Comment by afc163** [comment]
Could you provide a reproduce repository?

**Comment by vprada25** [comment]
![code](https://github.com/ant-design/ant-design/assets/41810525/3fcae295-5575-4653-915e-cbb61b7aaa19)

![code2](https://github.com/ant-design/ant-design/assets/41810525/254a8c2b-2da7-4c5a-baed-366969b9a39d)


at the moment it is not possible for me, but it happens when I import an icon directly from antd and use it in a switch

**Reply by afc163** [reply]
Could you reproduce it in https://codesandbox.io/ ?
